test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/compiler/resources/nugetSearchPage1.json

```diff
@@ -0,0 +1,5320 @@
+        },
+        {
+          "version": "4.6.1",
+          "downloads": 69818,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.6.1.json"
+        },
+        {
+          "version": "4.6.2",
+          "downloads": 20040,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.6.2.json"
+        },
+        {
+          "version": "4.6.3",
+          "downloads": 363,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.6.3.json"
+        },
+        {
+          "version": "4.6.4",
+          "downloads": 1073,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.6.4.json"
+        },
+        {
+          "version": "4.7.0-preview1-4986",
+          "downloads": 1560,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.0-preview1-4986.json"
+        },
+        {
+          "version": "4.7.0-preview4.5065+e27e4cab3fbb54b543cab8405cdfe82d2037fb59",
+          "downloads": 282,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.0-preview4.5065.json"
+        },
+        {
+          "version": "4.7.0-rtm.5104+9467a1c2030164f8dc28dfa601f126aade505d8f",
+          "downloads": 1251,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.0-rtm.5104.json"
+        },
+        {
+          "version": "4.7.0-rtm.5148+9245481f357ae542f92e6bc5e504fc898cfe5fc0",
+          "downloads": 4217,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.0-rtm.5148.json"
+        },
+        {
+          "version": "4.7.0+9245481f357ae542f92e6bc5e504fc898cfe5fc0",
+          "downloads": 110790,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.0.json"
+        },
+        {
+          "version": "4.7.2+857177140e83c5331632fefe336365030c366e3a",
+          "downloads": 53,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.2.json"
+        },
+        {
+          "version": "4.7.3+5f03bad3794603516c0a1f395f91d056246a1070",
+          "downloads": 51,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.7.3.json"
+        },
+        {
+          "version": "4.8.0-preview1.5156+d2efa5148f2644f86ea5b4e8da87e5a3c035d470",
+          "downloads": 1362,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.8.0-preview1.5156.json"
+        },
+        {
+          "version": "4.8.0-preview3.5278+c3240b16fcf3276246fc8c610771d14ab94fdc02",
+          "downloads": 282,
+          "@id": "https://api.nuget.org/v3/registration3-gz-semver2/nuget.versioning/4.8.0-preview3.5278.json"
+        },
+        {
+          "version": "4.8.0-rtm.5362+dc0b55f597c6224a13d64b984651b18700d3054a",
+          "do
```

Comment:
Can you use fake data instead? We shouldn't be store customer's data like this due to GDPR

Reply:
Discussed offline, I'll patch it up :)